request/src/main/java/com/vimeo/networking2/VimeoService.kt

```diff
@@ -0,0 +1,478 @@
+/*
```

Comment:
This looks like the only file with copyright, shouldn't it be applied to all files?